notebooks/stable-diffusion-v3/stable-diffusion-v3-torch-fx.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if not Path("sd3_helper.py").exists():
</pre><p>I do not see any imports from this module in notebook, is it just accidentally copied from another notebook?  The same about sd3_quantization_helper</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes, These were copied over accidentally. I will remove it and add <code>sd3_torch_fx_helper.py</code>.</p>